core/src/test/scala/com/rallyhealth/vapors/core/example/Snippets.scala

```diff
@@ -0,0 +1,57 @@
+package com.rallyhealth.vapors.core.example
+
+import cats.Id
+import com.rallyhealth.vapors.core.algebra.Expr
+import com.rallyhealth.vapors.core.data.{FactTable, FactType}
+import com.rallyhealth.vapors.core.dsl._
+
+import java.time.temporal.ChronoUnit
+import java.time._
+
+class Snippets(val clock: Clock) {
```

Comment:
Can we have a better name than Snippets?  AgeAndDobExpressions?

Reply:
I was thinking of this as similar to `example.FactTypes`... Maybe `example.FactExpressions`?